docs/benchmarks/test-results-summary.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Performance Benchmarks and Test Reports
+---
+
+
+## Introduction
+This page summarizes key performance metrics for nwaku and provides links to detailed test reports.
+
+> ## Quick reference
+>
+> - **10%** decrease in bandwidth usage in larger networks (1000 vs 2000 relay nodes)
+> - **0.4s** average, **5.5s** max message propogation latency (2000 relay nodes)
+> - Relevancy to Status App: TODO
+
+
+
+## Insights
+*Some metrics for specific protocols are currently unavailable due to reporting and logging limitations. Improvements are in progress.*
+
+### Bandwidth Usage: nwaku v0.34
+Average `libp2p` bandwidth usage for  1KB message size in a 1000 relay nodes network
```

Comment:
I think readers will just need a bit more guidance to understand what we mean by these terms, e.g. Whose bandwidth? What is the variable and what are the constants here?

An example suggestion:
Average `libp2p` per-node bandwidth usage for various message _injection_ rates into a Relay network of constant size `1000`. The messages are 1KB in size.